superset/assets/javascripts/explore/components/controls/BoundsControl.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Col, Row, FormGroup, FormControl } from 'react-bootstrap';
+import ControlHeader from '../ControlHeader';
+
+const propTypes = {
+  name: PropTypes.string.isRequired,
+  label: PropTypes.string,
+  description: PropTypes.string,
+  onChange: PropTypes.func,
+  value: PropTypes.array,
+};
+
+const defaultProps = {
+  label: null,
+  description: null,
+  onChange: () => {},
+  value: [null, null],
```

Comment:
i wonder if it would be more clear to store the min/max values as separate keys rather than an array. then we don't have to keep track of which value in the array is min or max.